src/Console/Command/AddPrefixCommand.php

```diff
@@ -234,4 +253,96 @@ private function validateOutputDir(InputInterface $input, OutputStyle $io)
             $this->fileSystem->remove($outputDir);
         }
     }
+
+    private function validateReplaceStrings(InputInterface $input)
+    {
+        $replaceStrings = $input->getOption(self::REPLACE_STRINGS_OPT);
+
+        if (null === $replaceStrings) {
+            $replaceStrings = $this->makeAbsolutePath(self::REPLACE_STRINGS_DEFAULT);
+
+            if (false === $this->fileSystem->exists($replaceStrings)) {
+                
+                return;
+            }
+        } else {
+            $replaceStrings = $this->makeAbsolutePath($replaceStrings);
+
+            if (false === $this->fileSystem->exists($replaceStrings)) {
+                throw new RuntimeException(
+                    sprintf(
+                        'Replacement strings config "<comment>%s</comment>" does not exist.',
+                        $replaceStrings
+                    )
+                );
+            }
+        }
+
+        $input->setOption(self::REPLACE_STRINGS_OPT, $replaceStrings);
+
+        if (false === is_readable($replaceStrings)) {
+            throw new RuntimeException(
+                sprintf(
+                    'Expected "<comment>%s</comment>" to be readable.',
+                    $replaceStrings
+                )
+            );
+        }
+
+        try {
+            $cwd = getcwd();
+            $fileSystem = $this->fileSystem;
+
+            $replaceStringsMap = Yaml::parse(
+                file_get_contents($replaceStrings)
+            );
+            if (empty($replaceStringsMap) || !isset($replaceStringsMap['replacements'])) {
+                throw new RuntimeException(
+                    sprintf(
+                        'No replacement strings configured in: %s', $replaceStrings
+                    )
+                );
+            }
+
+            $map = array_map(
+                function (array $replacement) use ($cwd, $fileSystem) {
+                    $replacement['files'] = array_map(
+                        function (string $path) use ($cwd, $fileSystem) {
+                            if (false === $fileSystem->isAbsolutePath($path)) {
+                                return $cwd.DIRECTORY_SEPARATOR.$path;
+                            }
+
+                            return $path;
+                        },
+                        $replacement['files']
+                    );
+
+                    return $replacement;
+                },
+                $replaceStringsMap['replacements']
+            );
+
+            return $map;
+        } catch (RuntimeException $e) {
+            throw $e;
+        } catch (\Exception $e) {
+            throw new RuntimeException(
+                sprintf(
+                    'Unable to parse replacement strings JSON file: %s%s%s',
+                    $replaceStrings,
+                    PHP_EOL,
+                    $e->getMessage()
+                )
+            );
+        }
+    }
+
+    private function makeAbsolutePath(string $path): string
```

Comment:
Ye olde extract method, should be propagated to a few methods in this class where this check is repeated.